.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
.vscode should be deleted :-)

Reply:
Thanks! Removed .vscode and added to .gitignore."